bot/modules/nostr/index.ts

```diff
@@ -0,0 +1,40 @@
+require('websocket-polyfill'); // is it needed?
+import { logger } from '../../../logger';
+import * as Config from './config';
+import { createOrderEvent } from './events';
+import * as Commands from './commands';
+import { Telegraf } from 'telegraf';
+import { MainContext } from '../../start';
+import { IOrder } from '../../../models/order';
+
+export const configure = (bot: Telegraf<MainContext>) => {
+  bot.command('/nostr', Commands.info);
+
+  if (!Config.getRelays().length) {
+    ['wss://nostr-pub.wellorder.net', 'wss://relay.damus.io'].map(
+      Config.addRelay
+    );
+  }
+
+  // I don't know why these requires are here and not at the top of the file, 
+  // so I leave them as they are instead of converting to imports.
```

Comment:
yes you can move it to the top of the file, there is a lot of line of code we can fix, but we are doing it gradually, when we see cases like this we fix it 😃

Reply:
converted to import and moved to the top